SPECS/dpdk/dpdk.spec

```diff
@@ -30,8 +30,7 @@
 Summary:        Set of libraries and drivers for fast packet processing
 Name:           dpdk
 Version:        18.11.2
-Release:        6%{?dist}
-Epoch:          2
```

Comment:
Are we sure we want to remove the epoch for packages already present in PMC?

Reply:
Yeah.  This should be fine because 1.0 to 2.0 is a breaking change.  I acknowledge that there might be an issue upgrading from 1.0 to 2.0 though. We can cross that bridge when we get there.  It will be a much larger issue than just this

Reply:
@PawelWMS can you please approve :)